.github/workflows/lint.yaml

```diff
@@ -33,3 +33,22 @@ jobs:
       - name: Fail if lint failed
         if: steps.linter.outputs.checks-failed > 0
         run: exit 1
+  py_lint:
+    name: Python format and lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.11
```

Comment:
We should probably set up 3.8 since that's the minimum version we support

Reply:
Done